front/.eslintrc.js

```diff
@@ -5,7 +5,14 @@ module.exports = {
     tsconfigRootDir: __dirname,
     sourceType: 'module',
   },
-  plugins: ['@typescript-eslint/eslint-plugin', 'unused-imports', 'simple-import-sort', 'twenty'],
+  plugins: [
+      '@typescript-eslint/eslint-plugin',
+      'unused-imports',
+      'simple-import-sort',
+      'twenty',
+      'eslint-plugin-twenty',
```

Comment:
We don't need this because we already have `twenty` that imports our package here.